bokehjs/test/playwright.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from '@playwright/test';
+export default defineConfig({
+  webServer: {
+    command: 'cd ../../docs/bokeh && make serve',
```

Comment:
Documentation is non-trivial and slow to build, thus this will get old very quickly when running on a regular basis. Further more this makes bokehjs and in result its CI dependent on documentation, which is what we don't want. Ultimately this will have to be probably moved to bokeh's tests and a CI job set up that depends on documentation CI job, so that we don't have to build docs twice.

Reply:
I think ideally there is some `node make` way to kick off these checks, and then the checks only actually get run in the docs job.